projects/common/vcu128/vcu128_system_bd.tcl

```diff
@@ -162,8 +175,8 @@ ad_connect axi_ddr_cntrl/c0_ddr4_ui_clk_sync_rst sys_500m_rstgen/ext_reset_in
 ad_connect axi_ddr_cntrl/c0_ddr4_ui_clk_sync_rst sys_hbm_rstgen/ext_reset_in
 ad_connect sys_mem_clk axi_ddr_cntrl/c0_ddr4_ui_clk
 ad_connect sys_mem_clk axi_ddr_cntrl_rstgen/slowest_sync_clk
-ad_connect sys_cpu_clk axi_ddr_cntrl/addn_ui_clkout1
-ad_connect sys_cpu_clk sys_rstgen/slowest_sync_clk
+ad_connect sys_mb_clk axi_ddr_cntrl/addn_ui_clkout1
+ad_connect sys_mb_clk sys_rstgen/slowest_sync_clk
```

Comment:
Make sure to have a two reset generators, one for the 200 MHz clock and one for the 100 MHz. 
In the current version **sys_cpu_reset** used for the 100 MHz domain is connected to the 200 MHz  clock.